ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/RollbackSpam.hs

```diff
@@ -0,0 +1,55 @@
+{-# LANGUAGE BlockArguments      #-}
```

Comment:
We removed it from all the other files but I don't remember if there was a specific reason? Like CI rejects it or something?